File: front/app/src/js/components/game/Scene/PaddleBoundingBox.js
export class PaddleBoundingBox {
  #y_min;
  #y_max;

  constructor(boardHeight, paddleHeight) {
    this.#y_max = boardHeight * 0.5 - paddleHeight * 0.5;
    this.#y_min = -this.#y_max;
  }

  get y_min() {
    return this.#y_min;
  }

  get y_max() {
    return this.#y_max;
  }
}

File: pong_server/src/game_server/Game/PaddleBoundingBox.py
import settings as settings


class PaddleBoundingBox(object):
    def __init__(self):
        self._y_max = settings.BOARD_SIZE[1] * 0.5 - settings.PADDLE_SIZE[1] * 0.5
        self._y_min = -self._y_max

    def get_y_min(self):
        return self._y_min

    def get_y_max(self):
        return self._y_max
